### PR TITLE
Fixes various missing texture errors

### LINKED ISSUE #40199: Missing tiles on shuttle
[Round ID]: # 93671
image

[For Admins]: #

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
fix: Fixed various missing texture errors.
/:cl:

Closes: #40199

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/turretedoutpost.dmm [hunk 1]
```diff
@@ -221,12 +218,6 @@
 /obj/machinery/door/airlock/public/glass,
 /turf/open/floor/plasteel/dark,
 /area/ruin/space/has_grav/turretedoutpost)
-"aJ" = (
-/turf/open/floor/plasteel{
-	dir = 4;
-	icon_state = "darkredfull"
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/turretedoutpost.dmm]: Used to have red turf decals on the floor here. Mind adding them?

Comment (ShizCalev) [reply _maps/RandomRuins/SpaceRuins/turretedoutpost.dmm]: image

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/RandomRuins/SpaceRuins/gasthelizards.dmm [hunk 2]
```diff
@@ -136,20 +136,14 @@
...
+/turf/open/floor/plasteel,
 /area/ruin/space/has_grav/gasthelizard)
 "q" = (
 /obj/machinery/door/window/brigdoor/northleft,
 /obj/machinery/atmospherics/components/binary/pump{
 	dir = 1
 	},
-/turf/open/floor/plasteel{
-	dir = 1;
-	icon_state = "blue"
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/gasthelizards.dmm]: image
decals

File: _maps/RandomRuins/SpaceRuins/gasthelizards.dmm [hunk 3]
```diff
@@ -136,20 +136,14 @@
 	dir = 1;
 	layer = 2.9
 	},
-/turf/open/floor/plasteel{
-	dir = 1;
-	icon_state = "blue"
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/gasthelizards.dmm]: .

**Review by ShizCalev — DISMISSED** [review]

File: _maps/RandomRuins/SpaceRuins/gasthelizards.dmm (hunk 2 above)
Comment (81Denton) [reply _maps/RandomRuins/SpaceRuins/gasthelizards.dmm]: Ah, that explains it. Give me a minute

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**